src/preload/main-window.js

```diff
@@ -71,24 +71,37 @@ const runtimeApi = {
 	setLocale: async (value) => {
 		return ipcRenderer.invoke('settings:set:locale', value)
 	},
+
+	// Sentry
+	getSentryUserId: () => {
+		return getProcessArgValue('comapeo-sentry-user-id')
+	},
+	enableSentryOnInit: () => {
+		return getProcessArgValue('comapeo-sentry-enable') === 'true'
+	},
 }
 
-function getAppVersion() {
-	const flag = process.argv
+/**
+ * @param {`comapeo-${string}`} flag
+ *
+ * @returns {string}
+ */
+function getProcessArgValue(flag) {
```

Comment:
why not use an existing parser like yargs? Or any other argv parser.

Reply:
for a long time, i didn't know that preload scripts had access to installed npm modules. can definitely switch to using a better module for this!

Reply:
they can even load native node modules, although the electron docs caution against this

Reply:
ah wait nevermind, I misread the documentation [here](https://www.electronjs.org/docs/latest/tutorial/tutorial-preload)

> On top of [Electron modules](https://www.electronjs.org/docs/latest/api/app), you can also access [Node.js built-ins](https://nodejs.org/dist/latest/docs/api/), as well as any packages installed via npm

Thought this section was referring to preload scripts but it's referring to the main process. By default with the sandboxing enabled, preload scripts don't have access to npm-installed modules, hence why I have this custom arg parser.

Really wish they'd expose some of `node:utils` in the preload scripts!

Reply:
> they can even load native node modules, although the electron docs caution against this

I think this assumes that the sandboxing is disabled? similar to my realization about normal npm modules. for now I'd rather not disable the sandboxing unless it's blocking essential functionality.